src/applications/_mock-form-ae-design-patterns/pages/contactInformation1.js

```diff
@@ -0,0 +1,42 @@
+// In a real app this would not be imported directly; instead the schema that
+// is imported from vets-json-schema should include these common definitions:
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+
+import * as address from 'platform/forms-system/src/js/definitions/address';
+import phoneUI from 'platform/forms-system/src/js/definitions/phone';
+
+// once the form schema is added to vets-json-schema, import this from there
+// and remove this duplicate
+import fullSchema from '../00-1234-schema.json';
```

Comment:
ahh so it looks like you do need that schema json file after all, or can we update this to remove it? Looks like it is failing to resolve here @bellepx0

Reply:
I removed this contactInformation1.js file since it is not needed